frontend/public/components/app-contents.tsx

```diff
@@ -413,12 +413,21 @@ const AppContents: React.FC<{}> = () => {
         exact
         kind="Route"
         loader={() =>
-          import('./routes/create-route' /* webpackChunkName: "create-route" */).then(
-            (m) => m.CreateRoute,
+          import('./routes/RoutePage' /* webpackChunkName: "create-route" */).then(
+            (m) => m.RoutePage,
           )
         }
       />
 
+      <LazyRoute
+        path="/k8s/ns/:ns/routes/:name/edit"
```

Comment:
to keep it consistent with the build config's edit form URL 
```suggestion
        path="/k8s/ns/:ns/routes/:name/form"
```